multichain-testing/test/stake-ica.test.ts

```diff
@@ -3,12 +3,29 @@ import type { TestFn } from 'ava';
 import { commonSetup, SetupContextWithWallets } from './support.js';
 import { makeDoOffer } from '../tools/e2e-tools.js';
 import { makeQueryClient } from '../tools/query.js';
-import { sleep } from '../tools/sleep.js';
+import { sleep, type RetryOptions } from '../tools/sleep.js';
 
 const test = anyTest as TestFn<SetupContextWithWallets>;
 
 const accounts = ['user1', 'user2'];
 
+/**
+ * Wait 90 seconds to ensure staking rewards are available.
+ *
+ * While we expect staking rewards to be available after a
+ * single block (~5-12 seconds for most chains), this provide additional
```

Comment:
```suggestion
 * single block (~5-12 seconds for most chains), this provides additional
 ```
 😮‍💨 not worth the ci run. 
 
 The const below doesn't need to be exported either.